zeabur部署最新版报错

**例行检查**

[//]: # (方框内删除已有的空格，填 x 号)
+ [x] 我已确认目前没有类似 issue
+ [x] 我已确认我已升级到最新版本
+ [x] 我已完整查看过项目 README，尤其是常见问题部分
+ [x] 我理解并愿意跟进此 issue，协助测试和提供反馈 
+ [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 issue 可能会被无视或直接关闭**

**问题描述**

```shell

#16 [builder2 4/7] RUN go mod download

#18 ...

#19 [builder2 5/7] COPY . .

#18 [builder  6/13] RUN npm install

#19 DONE 0.2s

#18 29.30 npm WARN 

#18 29.42 npm

deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead

 WARN deprecated rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser

#18 30.35 npm WARN deprecated workbox-cacheable-response@6.6.0: workbox-background-sync@6.6.0

#18 30.53 npm WARN deprecated workbox-google-analytics@6.6.0: It is not compatible with newer versions of GA starting with v4, as long as you are using GAv3 it should be ok, but the package is not longer being maintained

#18 30.73 npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility

#18 32.33 npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.

#18 32.46 npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead

#18 32.41 npm WARN deprecated domexception@2.0.1: Use your platform's native DOMException instead

#18 34.74 npm WARN deprecated @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead.

#18 34.74 npm WARN deprecated @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.

#18 34.74 npm WARN deprecated @babel/plugin-proposal-private-methods@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-private-methods instead.

#18 34.74 npm WARN deprecated @babel/plugin-proposal-numeric-separator@7.18.6: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-numeric-separator instead.

#18 35.04 npm WARN deprecated @babel/plugin-proposal-optional-chaining@7.21.0: This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-chaining instead.

#18 39.12 npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.

#18 45.03   run `npm fund` for details

#18 45.03 260 packages are looking for funding

#18 45.03 

#18 45.03 added 1516 packages, and audited 1517 packages in 45s

#18 45.03 

#18 45.04 npm notice 

#18 45.04 npm notice Run `npm install -g npm@10.8.0` to update!

#18 45.04 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.8.0>

#18 45.04 npm notice New major version of npm available! 8.19.4 -> 10.8.0

#18 45.04 npm notice 

#18 45.04 Run `npm audit` for details.

#18 45.04 

#18 45.04   npm audit fix --force

#18 45.04 To address all issues (including breaking changes), run:

#18 45.04 

#18 45.04 9 vulnerabilities (3 moderate, 6 high)

#18 45.04 

#18 DONE 53.2s

#20 [builder  7/13] RUN DISABLE_ESLINT_PLUGIN='true' REACT_APP_VERSION=$(cat VERSION) npm run build

#20 0.459 cat: VERSION: No such file or directory

#20 0.821 

#20 0.821 > react-scripts build && rm -rf ../build/default && mv -f build ../build/default

#20 0.821 > react-template@0.1.0 build

#20 0.821 

#20 2.370 Creating an optimized production build...

#20 5.377   

#20 5.377 go away.

#20 5.377 your devDependencies to work around this error. This will make this message

#20 5.377 ever be fixed. Add "@babel/plugin-proposal-private-property-in-object" to

#20 5.377 is not maintianed anymore. It is thus unlikely that this bug will

#20 5.377 babel-preset-react-app is part of the create-react-app project, which

#20 5.377 

#20 5.377 node_modules folder for unrelated reasons, but it may break at any time.

#20 5.377 "@babel/plugin-proposal-private-property-in-object" is already in your

#20 5.377 declaring it in its dependencies. This is currently working because

#20 5.377 "@babel/plugin-proposal-private-property-in-object" package without

#20 5.377 One of your dependencies, babel-preset-react-app, is importing the

#20 27.75 

#20 27.75 File sizes after gzip:

#20 27.75 

#20 27.75 Compiled successfully.

#20 27.82 

#20 27.82   https://cra.link/deployment

#20 27.82 

#20 27.82 Find out more about deployment here:

#20 27.82 

#20 27.82   serve -s build

#20 27.82   npm install -g serve

#20 27.82 

#20 27.82 You may serve it with a static server:

#20 27.82 The build folder is ready to be deployed.

#20 27.82 

#20 27.82 You can control this with the homepage field in your package.json.

#20 27.82 The project was built assuming it is hosted at /.

#20 27.82 

#20 27.82   755 B      build/static/js/585.3ffb3f2a.chunk.js

#20 27.82   2.4 kB     build/static/js/832.ab3d9d10.chunk.js

#20 27.82   98.49 kB   build/static/css/main.54631fc9.css

#20 27.82   158.89 kB  build/static/js/main.917e0036.js

#20 DONE 28.0s

#22 [builder  9/13] RUN npm install

#21 DONE 0.0s

#21 [builder  8/13] WORKDIR /web/berry

#22 5.752 npm ERR!

5.758 

5.756 npm ERR! See /root/.npm/eresolve-report.txt for a full report.

5.756 npm ERR! 

5.756 npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

5.756 npm ERR! this command with --force, or --legacy-peer-deps

5.756 npm ERR! Fix the upstream dependency conflict, or retry

5.756 npm ERR! 

5.756 npm ERR!   react-apexcharts@"^1.4.0" from the root project

 > [builder  9/13] RUN npm install:

------

#22 ERROR: process "/bin/sh -c npm install" did not complete successfully: exit code: 1

#22 5.758 npm ERR!     /root/.npm/_logs/2024-05-23T02_13_56_828Z-debug-0.log

#22 5.758 npm ERR! A complete log of this run can be found in:

#22 5.758 

#22 5.756 npm ERR! See /root/.npm/eresolve-report.txt for a full report.

#22 5.756 npm ERR! 

#22 5.756 npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

#22 5.756 npm ERR! this command with --force, or --legacy-peer-deps

#22 5.756 npm ERR! Fix the upstream dependency conflict, or retry

#22 5.756 npm ERR! 

#22 5.756 npm ERR!   react-apexcharts@"^1.4.0" from the root project

#22 5.756 npm ERR! node_modules/react-apexcharts

#22 5.756 npm ERR! peer apexcharts@"^3.41.0" from react-apexcharts@1.4.1

#22 5.756 npm ERR! Could not resolve dependency:

#22 5.756 npm ERR! 

#22 5.756 npm ERR!   apexcharts@"3.35.3" from the root project

#22 5.756 npm ERR! node_modules/apexcharts

#22 5.756 npm ERR! Found: apexcharts@3.35.3

#22 5.756 npm ERR! While resolving: one_api_web@1.0.0

#22 5.756 npm ERR! 

#22 5.756 npm ERR! ERESOLVE unable to resolve dependency tree

 code ERESOLVE

------

5.758 npm ERR!     /root/.npm/_logs/2024-05-23T02_13_56_828Z-debug-0.log

5.758 npm ERR! A complete log of this run can be found in:

error: failed to solve: process "/bin/sh -c npm install" did not complete successfully: exit code: 1

--------------------

  35 |     

  34 |     RUN DISABLE_ESLINT_PLUGIN='true' REACT_APP_VERSION=$(cat VERSION) npm run build

  33 | >>> RUN npm install

  32 |     WORKDIR /web/berry

  31 |     

--------------------

Dockerfile:33

Failed to build with zeaburpack: run buildctl build: exit status 1
```

**复现步骤**
最新版本创建 然后构建

**预期结果**
正常

7 |

38 |     WORKDIR /web/berry

39 | >>> RUN npm install

40 |     RUN DISABLE_ESLINT_PLUGIN='true' REACT_APP_VERSION=$(cat VERSION) npm run build

41 |

--------------------

error: failed to solve: process "/bin/sh -c npm install" did not complete successfully: exit code: 1

#22 1.882 npm ERR! See /root/.npm/eresolve-report.txt for a full report.

#22 1.885

#22 1.885 npm ERR! A complete log of this run can be found in:

#22 1.885 npm ERR!     /root/.npm/_logs/2024-05-27T14_13_54_262Z-debug-0.log

#22 ERROR: process "/bin/sh -c npm install" did not complete successfully: exit code: 1

--- a/web/berry/package.json
+++ b/web/berry/package.json
@@ -26,7 +26,7 @@
     "notistack": "^3.0.1",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
-    "react-apexcharts": "^1.4.0",
+    "react-apexcharts": "1.4.0",
     "react-device-detect": "^2.2.2",
     "react-dom": "^18.2.0",
     "react-perfect-scrollbar": "^1.5.8",

